README.md

```diff
@@ -0,0 +1,42 @@
+# cross_modality_query
+
+Server-side code for HuBMAP Cells API.
+[Python](https://github.com/hubmapconsortium/hubmap-api-py-client) and 
+[Javascript](https://github.com/hubmapconsortium/hubmap-api-js-client) clients available.
+
+## Usage
+
+This is only a high-level outline of the API:
+See the [Python client examples](https://github.com/hubmapconsortium/hubmap-api-py-client/tree/main/examples) for details.
+
+The API supports a small number of basic operations which can be combined to construct more complex queries.
+The API runs at a **`base_url`**: Currently `https://cells.dev.hubmapconsortium.org/api` is available.
+
+Issuing a `POST` to `{base_url}/{output_type}/` (where `output_type` is `cell`, `organ`, `gene`, or `cluster`),
+with query parameters in the body, will return a **query set handle**.
+(The Python and Javascript interfaces provide a **query set** abstraction, so you don't need to deal directly with the handle.)
+
+Issuing a `POST` to `{base_url}/{operation}/` (where `operation` is `union`, `intersection`, or `difference`),
+with query set handles provided as `key_one` and `key_two` in the body, will return a new query set handle,
```

Comment:
It seems that using `handle_` instead of `key_` would be more informative (but I understand that it might be too late for that).

Reply:
Chuck suggested this a little while ago, and I'm not opposed.  I just haven't gotten to it yet.